### ISSUE TITLE
(Request) Disable Virtual Keyboard Auto Complete

### ISSUE THREAD

**Issue by brentonv** [jurialmunkey/skin.arctic.zephyr.2#162]
Any chance of an option to disable Auto complete in the virtual keyboard? Only asking because it crashes the system. Thank you

**Comment by drinfernoo** [comment]
That's a known bug: https://github.com/phil65/plugin.program.autocompletion/issues/12

If it crashes on your install, just don't use it.

**Comment by brentonv** [comment]
Yeah I know it's a known issue was just hoping to turn it off as I can't uninstall it
